### PR TITLE
fix: VLAN reserve range side panels

### PR DESCRIPTION
**Pull request by petermakowski**
## Done
- fix: VLAN reserve range side panels
  - move SubnetActionForms.tsx out of the header directory

## QA steps

- [ ] Go to a VLAN details page (e.g. `/MAAS/r/vlan/35` in demos)
- [ ] Click reserve range
- [ ] Verify you can submit the form and range is added to the list
- [ ] Ensure you can edit and delete it successfully
- [ ] Add a new VLAN and ensure that Delete VLAN side panel works correctly

### CODE REVIEW

**Comment by ndv99** [comment]
I can delete a VLAN successfully, but the side panel does not close after this.

File: src/app/subnets/views/VLANDetails/VLANActionForms/VLANActionForms.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  activeForm,
+  setSidePanelContent,
+}: VLANActionFormProps) => {
+  return activeForm === VLANActionTypes.DeleteVLAN ? (
+    <VLANDeleteForm closeForm={() => setSidePanelContent(null)} id={id} />
+  ) : activeForm === VLANActionTypes.ReserveRange ? (
+    <ReservedRangeForm setActiveForm={setSidePanelContent} />
+  ) : activeForm === VLANActionTypes.DeleteReservedRange ? (
+    <ReservedRangeDeleteForm setActiveForm={setSidePanelContent} />
+  ) : null;
```
Comment (Jay-Topher) [inline src/app/subnets/views/VLANDetails/VLANActionForms/VLANActionForms.tsx:25]: We should probably represent this better, it's difficult to read

Comment (petermakowski) [reply src/app/subnets/views/VLANDetails/VLANActionForms/VLANActionForms.tsx]: You're right, wanted to limit the scope as it's a bugfix. Updated.

Comment (ndv99) [reply src/app/subnets/views/VLANDetails/VLANActionForms/VLANActionForms.tsx]: Agreed, a simple switch/case would be much more readable here

**Comment by ndv99** [comment]
Idk if this is just happening for me, but I'm still getting the issue with the "Delete reserved range" side panel not closing on success

**Pull request merged**